### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by AswathyVNair**
Home Page
homepage1

homepage2

### CODE REVIEW

File: pages/index.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            <button class ="btn-icon">
+                <a href="/"><i class="bi bi-heart-fill"></i></a>
+            </button>
+            <button class ="btn-icon">
+                <a href="/"><i class="bi bi-cart-fill"></i></a>
+            </button>
+        </div>
+    </nav>
+    <div class = "home-pg-container">
+
```
Comment (VishnuVNairCodes) [inline pages/index.html]: consider removing extra spaces

File: css/home.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+.img-card img{
+    opacity: 0.7;
+}
+
+.text-overlay{
+    position: absolute;
+    top:3rem;
+    left: 3rem;
+    color: var();
```
Comment (itsmebipul00) [inline css/home.css]: color is missing

File: css/home.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    padding: 1rem;
```
Comment (srishtigautam21) [inline css/home.css:45]: if you are not giving any color then you can delete this line

**Review by itsmebipul00 — APPROVED** [review]
Good to merge

**Review by srishtigautam21 — APPROVED** [review]
good to merge for now you can do the suggested changes in follow up PR.

File: pages/index.html [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+                <a href = "/">Shop Now</a>
+            </button>
+            
+        </div>
+
+        <h2>Popular Categories</h2>
+        <div class = "img-card-container">
+            
+            <div class="img-card">
+                <img src="/assets/nozzle.jpg">
```
Comment (Sahil25061999) [inline pages/index.html]: use alt for img tags

**Review by VishnuVNairCodes — APPROVED** [review]

**Review by Sahil25061999 — APPROVED** [review]
Rest of the code looks can be merged.

File: pages/index.html (hunk 1 above)
Comment (AswathyVNair) [reply pages/index.html]: Thanks for pointing that out. I had removed that.

File: css/home.css (hunk 2 above)
Comment (AswathyVNair) [reply css/home.css]: Thanks for pointing that out. I had made changes.

File: css/home.css (hunk 3 above)
Comment (AswathyVNair) [reply css/home.css:45]: Yes, I forgot to give color. Thanks for pointing that out.

**Review by Muzammil-khan710 — APPROVED** [review]
Good to merge

File: pages/index.html (hunk 4 above)
Comment (AswathyVNair) [reply pages/index.html]: Thanks for the suggestion.I have made changes

**Pull request merged**